examples/oft-solana/tasks/solana/debug.ts

```diff
@@ -114,23 +129,44 @@ task('lz:oft:solana:debug', 'Manages OFTStore and OAppRegistry information')
 
             DebugLogger.header('Checks')
             DebugLogger.keyValue('Admin (Owner) same as Delegate', oftStoreInfo.admin === delegate)
-            DebugLogger.keyValue('Token Mint Authority is OFT Store', unwrapOption(mintAccount.mintAuthority) === store)
+            DebugLogger.keyValue(
+                'Token Mint Authority is OFT Store',
+                unwrapOption(mintAccount.mintAuthority) === oftStore
+            )
             DebugLogger.separator()
         }
 
         const printPeerConfigs = async () => {
             const peerConfigs = dstEids.map((dstEid) => {
-                const peerConfig = oftDeriver.peer(store, dstEid)
+                const peerConfig = oftDeriver.peer(oftStore, dstEid)
                 return publicKey(peerConfig)
             })
+            const mockKeypair = new Keypair()
+            const point: OmniPoint = {
+                eid,
+                address: oftStore.toString(),
+            }
+            const endpointV2Sdk = new EndpointV2(
+                await createSolanaConnectionFactory()(eid),
+                point,
+                mockKeypair.publicKey // doesn't matter as we are not sending transactions
+            )
 
             DebugLogger.header('Peer Configurations')
+
             const peerConfigInfos = await oft.accounts.safeFetchAllPeerConfig(umi, peerConfigs)
-            dstEids.forEach((dstEid, index) => {
+            for (let index = 0; index < dstEids.length; index++) {
```

Comment:
Could be written more succinctly as:

```
for (const eid of dstEids) {
```

That way, you don't have to worry about the `index` being out of range as you are using an iterator instead.